robot_name : UAV1
vel : 1.0
yaw_rate : 0.15
arv_dist : 0.25
